data/nlu/nlu.md

```diff
@@ -1781,6 +1844,7 @@
 - what is rasa?
 - whats rasa
 - yeah go on explaining what rasa is
+- what are the components of rasa
```

Comment:
here i'm assuming that they mean "what are the different parts that make up rasa" and not NLU components i guess